package.json

```diff
@@ -171,4 +172,4 @@
 		"node": "^16 || ^18",
 		"npm": "^8 || ^9"
 	}
-}
+}
```

Comment:
Doesn't seem like we need to change this, as the next person (with it auto enabled) is probably going to add it again.